### PR TITLE
AWS IoT Demo

### PR DESCRIPTION
**Pull request by lthiery**
This is a simple demo that I'll be using tomorrow in the webinar. Hopefully we can land this today so that I can just copy paste from the repo during the webinar.

I've included the lambda function code and a README that provides the broad strokes of how things are done. Ohter than the Webinar, I'll do more detailed video and blog post soon hopefully.

### CODE REVIEW

**Review by kent-williams — COMMENTED** [review]
Updates to Readme still incoming?

File: ST-B-L072Z-LRWAN1/longfi-us915-button/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+This example demonstrates sending motion and environmental sensor data in [CayenneLPP](https://developers.mydevices.com/cayenne/docs/lora/#lora-cayenne-low-power-payload) format, using a B-L072Z-LRWAN1 development board with a X-NUCLEO-IKS01A3 expansion shield, to the myDevices Cayenne dashboard. For more information on adding your device to the Helium network, visit our quickstart guide [here](https://developer.helium.com/console/quickstart). For more information on adding your device to myDevices Cayenne, visit our guide [here](https://developer.helium.com/console/integrations/mydevices-cayenne-integration).
+
+## Required Arduino Libraries
+
+From the Arduino IDE, open the Library Manager (Sketch->Include Library->Manage Libraries). In the search box, type the library name below and install the latest version.
+
+[MCCI Arduino LoRaWAN Library](https://github.com/mcci-catena/arduino-lmic)  
+[CayenneLPP](https://github.com/ElectronicCats/CayenneLPP)  
+[X-NUCLEO-IKS01A3](https://github.com/stm32duino/X-NUCLEO-IKS01A3)
```
Comment (kent-williams) [inline ST-B-L072Z-LRWAN1/longfi-us915-button/README.md]: This isn't needed for this sketch right?

File: ST-B-L072Z-LRWAN1/longfi-us915-button/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+### B-L072Z-LRWAN1 - ST STM32L0 Discovery kit
+
+[B-L072Z-LRWAN1 Product Page](https://www.st.com/en/evaluation-tools/b-l072z-lrwan1.html)  
+[B-L072Z-LRWAN1 User Manual](https://www.st.com/content/ccc/resource/technical/document/user_manual/group0/ac/62/15/c7/60/ac/4e/9c/DM00329995/files/DM00329995.pdf/jcr:content/translations/en.DM00329995.pdf)  
+
+### X-NUCLEO-IKS01A3 - ST Motion MEMS and Environmental Sensor Board
+
+[X-NUCLEO-IKS01A3 Product Page](https://www.st.com/en/ecosystems/x-nucleo-iks01a3.html)  
+[X-NUCLEO-IKS01A3 User Manual](https://www.st.com/resource/en/user_manual/dm00601501-getting-started-with-the-xnucleoiks01a3-motion-mems-and-environmental-sensor-expansion-board-for-stm32-nucleo-stmicroelectronics.pdf)
```
Comment (kent-williams) [inline ST-B-L072Z-LRWAN1/longfi-us915-button/README.md]: +1

File: ST-B-L072Z-LRWAN1/longfi-us915-button/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+board = disco_l072cz_lrwan1
+framework = arduino
+
+lib_deps =
+     STM32duino LSM6DSO
+     STM32duino LIS2DW12
+     STM32duino STTS751
+     STM32duino LIS2MDL
+     STM32duino LPS22HH
+     STM32duino HTS221
```
Comment (kent-williams) [inline ST-B-L072Z-LRWAN1/longfi-us915-button/README.md]: +1

File: ST-B-L072Z-LRWAN1/longfi-us915-button/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+     MCCI LoRaWAN LMIC library
+     CayenneLPP
```
Comment (kent-williams) [inline ST-B-L072Z-LRWAN1/longfi-us915-button/README.md:73]: +1

**Comment by lthiery** [comment]
@kent-williams 
Oh I definitely messed up the README. I copied the wrong thing in. Will fix shortly

**Pull request merged**